index.html

```diff
@@ -5,22 +5,136 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Dashboard</title>
     <link rel="stylesheet" href="./styles/common.css">
+    <link rel="stylesheet" href="./styles/meals.css">
+    <link rel="stylesheet" href="./styles/stats.css">
     <link rel="stylesheet" href="./styles/dashboard.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100..900&display=swap" rel="stylesheet">
 </head>
 <body>
    
-    <div class="wrapper">
-        <div class="sidebar">
-            <ul>
-                <li>Meals</li>
-                <li>Report</li>
-                <li>Stats</li>
-            </ul>
+    <div class="navbar">
+        <div class="home-button">
+            <a href="../index.html"><button><img src="../images/dashboard_active.png" alt="dashboard" height="50px"></button></a>
+        </div>
+        <div class="menu-items">
+            <a href="./pages/diary.html"><button>Diary</button></a>
+            <a href="./pages/meals.html"><button >Meals</button></a>
+            <a href="./pages/recipes.html"><button>Recipes</button></a>
+            <a href="./pages/stats.html"><button>Stats</button></a>
         </div>
+    </div>
+
+    <div class="wrapper">
+
         <div class="content">
-            Dashboard
-            <a href="./pages/meals.html"> Go to meals</a>
+            <div class="dashboard-hero">
+                <div class="welcome-message">Welcome back, Boki</div>
+                <div class="user-image"><img src="./images/profile.jpeg" alt="Profile picture"></div>
+                <div class="user-streak">
+                    <ul>
+                        <li class="active-streak">M</li>
+                        <li class="active-streak">T</li>
+                        <li class="inactive-streak">W</li>
+                        <li class="active-streak">T</li>
+                        <li class="active-streak">F</li>
+                        <li class="active-streak">S</li>
+                        <li class="inactive-streak">S</li>
+                    </ul>
+                </div>
+                <div class="hero-background"></div>
+            </div>
+            
+            <!-- Today's Meal -->
+            <div class="weekly-meals-section">
+                <div class="meal-heading">
+                    <div class="weekday-name">Today's Meal Plan</div>
+                </div>
+                <div class="weekly-meals-content">
+                    <div class="recipe-container">
+                        <div class="rec-img"></div>
```

Comment:
it's better not to use shortcuts. I mean recipe => rec. 
We use it only for well know words (I mean well known in industry). For example: button => btn.
But it's better not do like this for domain specific. 
There is no need to fix it, just to be aware

Reply:
I've updated it to avoid shortcuts to follow best practice.